app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -180,6 +180,22 @@ export class MergeConflictsDialog extends React.Component<
   ): JSX.Element | null {
     switch (conflictStatus.kind) {
       case 'text':
+        if (conflictStatus.conflictMarkerCount === null) {
+          return (
+            <li className="unmerged-file-status-conflicts">
```

Comment:
:art: we've now repeated the same `<li>` and `<Octicon>` elements three times in here - any interest in lifting this up into `renderUnmergedFile` to avoid accidentally forgetting to change a `className` in here? e.g.

```
const content = this.renderConflictedFile(
  file.path,
  file.conflictStatus,
  () =>
    this.props.openFileInExternalEditor(
      join(this.props.repository.path, file.path)
    )
)

return (
  <li className="unmerged-file-status-conflicts">
    <Octicon symbol={OcticonSymbol.fileCode} className="file-octicon" />
    {content}
  </li>
)
```

Reply:
totally into this, was thinking the same thing :)